packages/ui-design-system/src/MenuWithCombobox/MenuWithCombobox.tsx

```diff
@@ -0,0 +1,200 @@
+import * as Ariakit from '@ariakit/react';
```

Comment:
is this an alternative to radix ?

Reply:
Yes, TBH I prefer a little bit the simplicity of Radix (composition first with hidden complexity) but Radix lag behind in term of new features and customization is sometimes really hard.

With that new pkg it's simpler (and there is a lot of concrete examples to allow complex components creation). I already used it for the SelectWithCombobox.

As an example, Radix don't have yet any combobox, but the open issue on the topic started 3/4 years ago. This is definitely a pain point for me. Composition of existing components is hard in radix but easier on Ariakit